README.md

```diff
@@ -158,7 +163,7 @@ We will set up a deploy key in the Git server that will allow an SSH key we make
     puppet agent -t
     ~~~
 
-9. Code Manager is configured and has been used to deploy your code
+Code Manager is configured and has been used to deploy control-repo's code.
```

Comment:
They don't actually do anything here, but it's worth saying they're done. Took out the step number.